.lintstagedrc

```diff
@@ -0,0 +1,12 @@
+{
+  "*.css": [
+    "prettier --write",
+    "git add",
+    "stylelint"
+  ],
+  "*.js": [
+    "prettier --write",
+    "git add",
```

Comment:
Ah ok. I've not had any problems with it thus far. I've added it because otherwise you'll have to manually run prettier, which is annoying. And using prettier is quite convenient for a repository like this, as otherwise you can easily get multiple code-styles in the same repo. It completely eliminates the problem.

Reply:
Yeah, like I said normally I'd expect code styles to be enforced later in the process, by the CI server. Auto-fixing style issues on commit sounds nice in theory though, so I'm willing to give it a shot.

Reply:
Cool 👍